modules/compliance-supported-profiles.adoc

```diff
@@ -8,86 +8,106 @@
+|`x86_64`
 
 |ocp4-moderate
 |NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift - Platform level
 |0.1.39+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
+|`x86_64`
+
+|rhcos4-e8
+|Australian Cyber Security Centre (ACSC) Essential Eight
+|0.1.39+
+|link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
+|`x86_64`
+
+|rhcos4-moderate
+|NIST 800-53 Moderate-Impact Baseline for Red Hat Enterprise Linux CoreOS
+|0.1.39+
+|link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
+|`x86_64`
 
 |ocp4-moderate-node
 |NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift - Node level
 |0.1.44+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
+|`x86_64`
 
 |ocp4-nerc-cip
 |North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for the Red Hat OpenShift Container Platform - Platform level
 |0.1.44+
 |link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
+|`x86_64`
 
 |ocp4-nerc-cip-node
 |North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for the Red Hat OpenShift Container Platform - Node level
 |0.1.44+
 |link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
+|`x86_64`
+
+|rhcos4-nerc-cip
+|North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for Red Hat Enterprise Linux CoreOS
+|0.1.44+
+|link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
+|`x86_64`
 
 |ocp4-pci-dss
 |PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4
 |0.1.47+
 |link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
+|`x86_64`
 
 |ocp4-pci-dss-node
 |PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4
 |0.1.47+
 |link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
-
-|rhcos4-e8
-|Australian Cyber Security Centre (ACSC) Essential Eight
-|0.1.39+
-|link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
-
-|rhcos4-moderate
-|NIST 800-53 Moderate-Impact Baseline for Red Hat Enterprise Linux CoreOS
-|0.1.39+
-|link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
-
-|rhcos4-nerc-cip
-|North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for Red Hat Enterprise Linux CoreOS
-|0.1.44+
-|link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
+|`x86_64`
```

Comment:
With the latest release of the compliance operator with 4.12, the ppc64le architecture supports PCI

Reply:
Latest upstream release (0.1.56)? Or downstream 0.1.53?

Reply:
0.1.56 on the upstream.

Reply:
Excellent - thanks for clarifying!

Reply:
Thanks @prb112 , I can include that when version 0.1.56 is released downstream.

Reply:
We're working on landing https://github.com/ComplianceAsCode/compliance-operator/pull/153 and we'll be releasing 0.1.57 upstream.